exercises11/main.js

```diff
@@ -0,0 +1,124 @@
+// exercise 1: Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+//cách 1
+function returnArray (charRepeat, numberRepeat) {
```

Comment:
Tên hàm đặt sát nghĩa vào
```suggestion
function repeat (word, times) {
```